json.go

```diff
@@ -22,6 +22,20 @@ type Document struct {
 	Status    string `json:"status"`
 }
 
+type Guide struct {
```

Comment:
To use it, you would have to generalise most of the shared stuff (e.g. `parse.go`, `rewrite.go`, `api.go`) - does that sound appealing in Go? :wink: